releasenotes/notes/meas-esp-5789e8cc0de39323.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add support for the ``use_measure_esp`` flag, which defines whether to use excited state
+    promoted (ESP) readout for the last measurement in each circuit.
```

Comment:
```suggestion
    Add support for the ``use_measure_esp`` flag in the
    :meth:`qiskit.providers.ibmq.IBMQBackend.run` method, which defines whether to 
    use excited state promoted (ESP) readout for the last measurement in each circuit.
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-ibmq-provider/pull/952/commits/42acb671b2ea57634e5bfef2849800cc4540be7f.